ArcGISRuntimeSDKQt_CppSamples/EditData/EditAndSyncFeatures/EditAndSyncFeatures.cpp

```diff
@@ -182,17 +182,17 @@ SyncGeodatabaseParameters EditAndSyncFeatures::getSyncParameters()
 void EditAndSyncFeatures::generateGeodatabaseFromCorners(double xCorner1, double yCorner1, double xCorner2, double yCorner2)
 {
   // create an envelope from the QML rectangle corners
-  auto corner1 = m_mapView->screenToLocation(xCorner1, yCorner1);
-  auto corner2 = m_mapView->screenToLocation(xCorner2, yCorner2);
-  auto extent = Envelope(corner1, corner2);
-  auto geodatabaseExtent = GeometryEngine::project(extent, SpatialReference::webMercator());
+  Point corner1 = m_mapView->screenToLocation(xCorner1, yCorner1);
+  Point corner2 = m_mapView->screenToLocation(xCorner2, yCorner2);
+  Envelope extent(corner1, corner2);
+  Geometry geodatabaseExtent = GeometryEngine::project(extent, SpatialReference::webMercator());
```

Comment:
These can all be const. It's optional since geometries are all immutable anyway.